hydra-filters/src/main/java/com/addthis/hydra/data/filter/bundle/BundleFilterTimeRange.java

```diff
@@ -114,10 +114,12 @@ public BundleFilterTimeRange(@JsonProperty(value = "time", required = true) Auto
             tafter = 0;
         }
 
-        if ( timeZone != null ) {           // regardless of timeFormat
-            this.format = DateTimeFormat.forPattern(timeFormat).withZone(DateTimeZone.forID(timeZone));
-        } else if( timeZone == null && timeFormat != null ) {
+        if( timeFormat != null && timeZone == null ) {
             this.format = DateTimeFormat.forPattern(timeFormat);
+        } else if ( timeFormat != null && timeZone != null ) {
+            this.format = DateTimeFormat.forPattern(timeFormat).withZone(DateTimeZone.forID(timeZone));
+        } else if ( timeFormat == null ) {
+            this.format = null;
         } else {
```

Comment:
This could be reduced to less if statements.

if tf == null;
else if tz == null;
else;